src/sections/dragdrop/DragDropBoard.tsx

```diff
@@ -158,7 +156,31 @@ const [boardSections, setBoardSections] =
     ...defaultDropAnimation,
   };
 
-  const ticket = activeTicketId ? getTicketById(tickets, activeTicketId) : null;
+  const getItemById = (tickets: Ticket[], id: number) => {
+    return tickets.find((ticket) => ticket.id === id);
+  };
+
+  const ticket = activeTicketId ? getItemById(items, activeTicketId) : null;
+
+  const initializeBoard = (tickets: Ticket[]) => {
+    const boardSections: BoardSections = {};
+  
+    Object.keys(categories).forEach((boardSectionKey) => {
+      boardSections[boardSectionKey] = getItemsByCategory(
+        tickets,
+        boardSectionKey
+      );
+    });
+  
+    return boardSections;
+  };
+
+  const getItemsByCategory = (tickets: Ticket[], status: String) => {
+    return tickets.filter((ticket) => ticket.status === status);
+  };
+
+  
+  
 
   return (
     boardSections &&
```

Comment:
After looking more closely at this, using a table would be better.  A table would not require predefining the width like a grid does.